docs/upgrade.html

```diff
@@ -95,6 +95,7 @@ <h5><a id="upgrade_200_notable" href="#upgrade_200_notable">Notable changes in 2
         timeout behavior for blocking APIs. In particular, a new <code>poll(Duration)</code> API has been added which
         does not block for dynamic partition assignment. The old <code>poll(long)</code> API has been deprecated and
         will be removed in a future version.</li>
+    <li>The implementation of DeleteRecordsCommand makes use of the new Java-based AdminClient.</li>
```

Comment:
I meant that the note should be about removing `AdminClient.deleteRecordsBefore`.